lib/new_relic/agent/instrumentation/active_merchant.rb

```diff
@@ -34,4 +34,19 @@ class ActiveMerchant::Billing::Gateway
       end
     end
   end
+
+  executes do
+    if ActiveMerchant::VERSION < Gem::Version.new('1.65.0')
```

Comment:
If this were a method we'd do something like this to avoid the giant whole-method `if`:

```ruby
def executes
  return unless ActiveMerchant::VERSION < Gem::Version.new('1.65.0')

  # deprecation warning
end
```

We can pull off the same thing with blocks by using `next`:

```ruby
executes do
  next unless ActiveMerchant::VERSION < Gem::Version.new('1.65.0')

  # deprecation warning
end
```

Reply:
Thank you! I'll make this adjustment to all the version conditions :)